sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -576,6 +576,13 @@ class Analyzer(
     /** Extracts a [[Generator]] expression and any names assigned by aliases to their output. */
     private object AliasedGenerator {
       def unapply(e: Expression): Option[(Generator, Seq[String])] = e match {
+        case Alias(g: Generator, name) if g.elementTypes.size > 1 => {
```

Comment:
Yeah, that's what I was thinking.
On May 18, 2015 6:00 PM, "Cheng Hao" <notifications@github.com> wrote:

> In
> sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala
> <https://github.com/apache/spark/pull/6178#discussion_r30561456>:
>
> > @@ -576,6 +576,13 @@ class Analyzer(
> >      /** Extracts a [[Generator]] expression and any names assigned by aliases to their output. */
> >      private object AliasedGenerator {
> >        def unapply(e: Expression): Option[(Generator, Seq[String])] = e match {
> > +        case Alias(g: Generator, name) if g.elementTypes.size > 1 => {
>
> Sorry, I am not sure what you mean for regex? You mean to check if the
> name like _c0, _c1 in regex?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/6178/files#r30561456>.
>